storybook/stubs/AppLayouts/Chat/stores/qmldir

```diff
@@ -1,3 +1,4 @@
+CreateChatPropertiesStore.qml
```

Comment:
this little thing causes tests segfaults :rofl:

Reply:
OMG :scream: